src/utils/link/link-by-prompt.js

```diff
@@ -132,25 +123,19 @@ Run ${chalk.cyanBright('`git remote -v`')} to see a list of your git remotes.`)
       }
 
       if (sites.length === 0) {
-        context.error(`No sites found named ${siteName}`)
+        context.error(`You don't have any sites. Use netlify 'sites:create' to create a site.`)
```

Comment:
At some point we could even consider letting the user pick a prompt to create a new site directly here.

Reply:
That's a great idea.